addons/account/models/account_tax.py

```diff
@@ -1582,3 +1582,15 @@ def _get_aml_target_tax_account(self, force_caba_exigibility=False):
             return self.tax_id.cash_basis_transition_account_id
         else:
             return self.account_id
+
+    def action_open_taxes(self, res_ids, title=None):
```

Comment:
@qdp-odoo Do you agree on having these methods on all the models?  It would avoid to have to re-create them in every module like we have now in https://github.com/odoo/enterprise/blob/49c9dc9151435fddfc2e3b62ad397015f144a7b6/account_reports/wizard/account_report_file_download_error_wizard.py#L53

Reply:
having the title as parameter seems flexible enough to me. I'm ok with that :+1:  :)